#exception Handling
print("Program start")
name=None
list1=[2,3]
try:
    #ans=int("1u")
    l=len(name)
    #elements=list1[5]
except Exception as e:
    print(e)
    

    
print("Program finish")
